backend/routes/auth.js

```diff
@@ -13,7 +13,7 @@ module.exports = di => {
 
     router.post('/login', [loginValidator, validate], (...args) => loginController.invoke(...args));
     router.post('/register', [registerValidator, validate], (...args) => registerController.invoke(...args));
-    router.post('/logout', (...args) => logoutController.invoke(...args));
+    router.get('/logout', (...args) => logoutController.invoke(...args));
```

Comment:
It should be POST

https://stackoverflow.com/questions/3521290/logout-get-or-post

Reply:
done